app/jobs/delete_missed_reservations_job.rb

```diff
@@ -0,0 +1,36 @@
+class DeleteMissedReservationsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
+    return if disabled
+    run
```

Comment:
I think these method names can be a *little* more descriptive :stuck_out_tongue:, for example, I would replace `run` with `delete_missed_reservations` and clean with `clean_up`, but overall I think that this is **much** better!